src/components/ActivityFeed/PostReactions.tsx

```
@@ -0,0 +1,200 @@
+							alt='Voter Profile'
+							className='h-6 w-6 rounded-full'
+						/>
+						<div className='text-[10px] md:ml-2 md:pt-5 md:text-[12px]'>
+							{reactionState.likesCount === 1 ? (
+								<p className='-mt-2'>{`${displayUsername} has liked this post`}</p>
+							) : (
+								<Popover
+									placement='bottom'
+									trigger='hover'
+									content={<>{renderUsernames('👍')}</>}
+									arrow={true}
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)